src/components/TextField/passwordField.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+
+interface PasswordField {
+  readonly name: string
+  readonly label: string
+  readonly placeholder: string
+  readonly errorInput: string
+}
+
+const PasswordField = ({ label, placeholder, errorInput }: PasswordField) => {
+  const [isPasswordShown, setPasswordShown] = useState(false)
+
+  const handlePasswordVisiblity = () => {
+    setPasswordShown(!isPasswordShown)
+  }
+
+  const renderValidInput = () => (
+    <>
+      <input
+        className="form__input"
+        type={isPasswordShown ? 'text' : 'password'}
```

Comment:
Since we prefer not to handle logic inside element props. Do you think it looks cleaner?
If you agree so, please change others also

```suggestion
    const inputType: string = isPasswordShown ? 'text' : 'password'
    return (<>
      <input
        className="form__input"
        type={inputType}
```